src/main/java/readAndWrite/ReadAndWrite.java

```diff
@@ -0,0 +1,108 @@
+package readAndWrite;
+
+import java.io.*;
+import java.util.*;
+
+public class ReadAndWrite implements Map<Integer, MenuSingleRow> {
+
+    private static Map<Integer, MenuSingleRow> readRow = new LinkedHashMap<>();
+    static {
+        try (BufferedReader bufferedReaderMenuRows = new BufferedReader(new FileReader("menuRows.txt"))) {
+
+            String userInput;
+            while ((userInput = bufferedReaderMenuRows.readLine()) != null) {
+                String[] data = userInput.split(",");
+                int rowNum = Integer.parseInt(data[0]);
+                String rowDescription = data[1];
+                Map<String, Integer> tempOption = new HashMap<>();
```

Comment:
Why don't you create some class here which has string and integer fields instead of this map?